### DISCUSSION TITLE
What's the difference between PrimeVue templates?

### DISCUSSION THREAD

**Discussion by petr-panek** [primefaces/community#2901] [PrimeVue Templates] [answered]
I'm building first project with PrimeVue, no experiance with PrimeVue yet. 
I'd like to understand what are differences between each PrimeVue templates?
Is there any guide how to choose "proper" template suitable to app type?

**Comment by cagataycivici** [comment, marked as answer]
Main difference is the design and customization features, underlying implementation is more or the less the same.
